apps/voting/app/package.json

```diff
@@ -4,16 +4,19 @@
   "private": true,
   "license": "AGPL-3.0-or-later",
   "dependencies": {
-    "@aragon/client": "^1.1.0",
+    "@aragon/api": "^1.0.0-beta.1",
+    "@aragon/api-react": "^1.0.0-beta.1",
     "@aragon/ui": "^0.31.0",
     "bn.js": "^4.11.8",
     "date-fns": "2.0.0-alpha.22",
     "onecolor": "^3.1.0",
-    "prop-types": "^15.6.0",
-    "react": "^16.5.2",
-    "react-dom": "^16.2.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
     "react-linkify": "^0.2.2",
     "react-spring": "^5.7.2",
+    "rxjs": "^6.2.1",
+    "rxjs-compat": "^6.2.1",
```

Comment:
I don't think we need the `rxjs-compat` given the changes in this PR.

https://github.com/ReactiveX/rxjs/blob/master/docs_app/content/guide/v6/migration.md#drop-compat